court/tests.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from court.models import Court
+from decimal import Decimal
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+
+
+@pytest.mark.django_db
+class TestCourtModel:
+
+    @pytest.fixture
+    def court(self):
+        court = Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
```

Comment:
Following @Yuval-Vino's comment, do you need the court ID explicitly defined here? (also applies to `test_create_court_with_existing_id` and `test_create_valid_court`)

Reply:
Also, what is the courtID value if you don't explicitly define it? are we sure that it will be auto-created with a unique value? 
Django default has this hyperparameter which is missing here `auto_created=True`. 
It's possible that the default value of auto_created is true but I think we need to check it

Reply:
I've deleted the courtID from other tests, and these went pretty good. I forgot about these functions that you've commented on. Unfortunately I have some errors when deleting the courtID, I'll make it work today.
@Yuval-Vino I've seen on a comment on StackOverflow mentioning that it's recommended to not use this hyperparameter, but the comment was from 2015 so I guess it's irrelevant. 
I'll fix these functions today. Thank you guys for your comments.